scanner/src/main/kotlin/Scanner.kt

```diff
@@ -72,7 +72,7 @@ import org.ossreviewtoolkit.utils.spdx.toSpdx
 const val TOOL_NAME = "scanner"
 
 @Suppress("TooManyFunctions")
-class Scanner(
+data class Scanner(
```

Comment:
Why `data class`?

Reply:
To be able to `copy()` the scanner with a different `scannerWrappers`: https://github.com/bosch-io/oss-review-toolkit/blob/b90e2a3b35ba8241be1d0d6d9c140d2a2cdebb71/scanner/src/funTest/kotlin/scanners/MultipleScannersTest.kt#L38

Reply:
I'm not sure about the data class thing, have you considered adding a parameter to `createScanner()` as alternative?